packages/backend/src/ee/models/AiAgentModel.ts

```diff
@@ -378,7 +376,8 @@ export class AiAgentModel {
                 const [result] = await trx(AiAgentInstructionVersionsTableName)
                     .insert({
                         ai_agent_uuid: agent.ai_agent_uuid,
-                        created_at: updatedAt,
+                        // @ts-expect-error - knex types are not correct
+                        created_at: trx.fn.now(),
```

Comment:
is this going to be the same `now` as the one from agent's `updated_at` value? [Docs](https://knexjs.org/guide/utility.html#now) don't say anything about this, which makes me think it's just a `now` utility
Just asking because this will have a slightly different value than the one from updated.

Reply:
this is going to use PG's NOW() function instead of js new Date()
do they have to match?

Reply:
it's not required that they match, but from previous experience, it's nice to have all the same timestamps when updating values in the db, specially when debugging incidents in one year 😅

Reply:
niiiice then we are good to go!